lili.json

```diff
@@ -0,0 +1,23 @@
+{
+    "version": "2.9.4",
+    "license": "GPL",
+    "url": "https://www.linuxliveusb.com/downloads/?version=stable-portable#/dl.7z",
+    "homepage": "https://www.linuxliveusb.com",
+    "extract_dir": "LinuxLive USB Creator 2.9.4",
+    "bin": "LiLi USB Creator.exe",
```

Comment:
What about also adding an alias without spaces?:
```
"bin": [
  "LiLi USB Creator.exe",
  [
    "LiLi USB Creator.exe",
    "LiLiUSBCreator"
  ]
],
```

Reply:
Excellent idea. I added the alias as just 'lili' to keep it short and sweet. Thank you! :)